Delegate_CategorySampleApp/ViewController.m

```diff
@@ -0,0 +1,78 @@
+//
+//  ViewController.m
+//  Delegate_CategorySampleApp
+//
+//  Created by webonise on 21/09/16.
+//  Copyright © 2016 webonise. All rights reserved.
+//
+
+#import "ViewController.h"
+
+@interface ViewController ()
+
+@end
+
+@implementation ViewController
+CustomAlertView *customAlertView;
+UIView *view1;
+- (void)viewDidLoad {
```

Comment:
- no need to take view1, and its bad naming convention, does not intend anything view1